src/components/WalletModal/WalletModal.tsx

```diff
@@ -102,7 +101,7 @@ export const WalletModal: React.FC = () => {
                   />
                 </Tabs.TabPane>
                 <Tabs.TabPane tab="Tokens" key="2" style={{ overflow: 'auto' }}>
-                  <TokensTab tokenList={tokenList} />
+                  <TokensTab assets={fromAssets} />
```

Comment:
you can move assets$ observable connection to TokensTab component. It will be better for performance

Reply:
moved assets$ into TokensTab.tsx,